app/logic/Calendar/ActiveSync/ActiveSyncEvent.ts

```diff
@@ -127,7 +127,7 @@ export class ActiveSyncEvent extends Event {
         MonthOfYear: this.recurrenceRule.frequency == Frequency.Yearly ? String(this.recurrenceRule.seriesStartTime.getMonth() + 1) : [],
         Until: toCompact(this.recurrenceRule.seriesEndTime) || [],
         DayOfMonth: [Frequency.Monthly, Frequency.Yearly].includes(this.recurrenceRule.frequency) && !this.recurrenceRule.week ? String(this.recurrenceRule.seriesStartTime.getDate()) : [],
-        FirstDayOfWeek: this.recurrenceRule.frequency == Frequency.Weekly ? String(this.recurrenceRule.first) : [],
+        FirstDayOfWeek: this.recurrenceRule.frequency == Frequency.Weekly && this.calendar.account.protocolVersion != "14.0" ? String(this.recurrenceRule.first) : [],
```

Comment:
This is logically correct but I don't like the condition ordering; I would prefer the version test to be first, or even to be a separate ternary, like this:
```suggestion
        FirstDayOfWeek: this.calendar.account.protocolVersion == "14.0" ? [] : this.recurrenceRule.frequency == Frequency.Weekly ? String(this.recurrenceRule.first) : [],
```